mne_qt_browser/tests/test_pg_specific.py

```diff
@@ -20,6 +20,26 @@
 SHOW_PROJECTORS = "Show projectors"
 
 
+def test_scalings(raw_orig, pg_backend):
+    """Test the correct parsing of the scalings dict to Text Boxes"""
```

Comment:
Our docstring formatting checker is very picky (it requires full sentences with period)

```suggestion
    """Test the correct parsing of the scalings dict to Text Boxes."""
```